docs/quickstart/docker.md

```diff
@@ -17,18 +17,26 @@ Coder with Docker has the following advantages:
 
     The Coder server binds to port 3000 by default. Use `--address :<port>` to customize it!
 
+    Use the [Coder tunnel](../admin/configure.md#tunnel) for a public URL:
+
+    ```bash
+    coder server
+    ```
+
+    Or set an [access URL](../admin/configure.md#access-url) to run entirely locally:
+
     ```bash
-    $ coder server
+    coder server --access-url=http://localhost:3000 --address=:3000
```

Comment:
@bpmct the `--address` flag missing the IP address, unless I'm missing something?

Reply:
Oddly, it doesn't need it.